documentation/src/main/asciidoc/topics/ref_indexing_configuration.adoc

```diff
@@ -144,13 +144,13 @@ include::xml/indexing_index_writer.xml[]
 |Maximum amount of memory that can be used for buffering added entries and deletions before they are flushed to the index storage. Large values result in faster indexing but use more memory. For faster indexing performance you should set this attribute instead of `max-buffered-entries`. When used in combination with the `max-buffered-entries` attribute, a flush occurs for whichever event happens first.
 
 |`thread-pool-size`
-|Number of threads that execute write operations to the index.
+|This configuration from Infinispan 15 is not used anymore, since the index engine uses the Infinispan thread pools.
```

Comment:
This configuration is ignored since Infinispan 15.0. The indexing engine now uses the Infinispan thread pools.

Reply:
Applied thanks